Lib/typing.py

```diff
@@ -1241,6 +1241,7 @@ def _alias(origin, params, inst=True):
 AsyncContextManager = _alias(contextlib.AbstractAsyncContextManager, T_co)
 Dict = _alias(dict, (KT, VT), inst=False)
 DefaultDict = _alias(collections.defaultdict, (KT, VT))
+OrderedDict = _alias(collections.OrderedDict, (KT, VT))
```

Comment:
This would require tests in `Lib/test/test_typing.py` probably along the lines of `test_chainmap_instantiation` and a [NEWS entry](https://devguide.python.org/committing/#what-s-new-and-news-entries)